api/src/main/java/com/intern/api/controller/MongoController.java

```diff
@@ -18,18 +20,35 @@
 
```

Comment:
Please divide your controller on less general components. For example: MessageController and UserController.